deploy/base1ns-gotemplate/ns_dev.yaml

```
@@ -0,0 +1,284 @@
+        modelmesh-enabled: "true"
+        name: '{{.SPACE_NAME}}-dev'
+        opendatahub.io/dashboard: "true"
+      name: '{{.SPACE_NAME}}-dev'
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      name: exec-pods
+      namespace: '{{.SPACE_NAME}}-dev'
+    rules:
+    - apiGroups:
+        - ""
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)